biztalk/esb-toolkit/install-and-configure-the-microsoft-biztalk-esb-toolkit.md

```diff
@@ -102,7 +102,7 @@ Starting with BizTalk Server 2013 and newer versions, [!INCLUDE[esbToolkit](../i
   
 ## Upgrade ESB Toolkit – Community Addition
```

Comment:
Remove this entire section as I didn't find any reliable info about in-place upgrade to the ESB Toolkit 2.2:

```suggestion
## Upgrade ESB Toolkit – Community Addition
```